backend/package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "NODE_ENV=prod node server.js",
+    "start": "yarn compile_server && NODE_ENV=prod node server-tsc/server.js",
     "start_db": "docker run -d --name buckets_db -p 27017:27017 -p 28017:28017 -e MONGODB_USER=\"username\" -e MONGODB_DATABASE=\"buckets_test\" -e MONGODB_PASS=\"password\" -e AUTH=no tutum/mongodb",
-    "build_server": "nodemon --exec 'node server.js'",
-    "start_server": "NODE_ENV=dev nodemon --exec 'node server.js'",
-    "test_server": "yarn start_test_server & NODE_ENV=test yarn compile_tests && NODE_ENV=test mocha --opts mocha.opts",
-    "start_test_server": "NODE_ENV=test node server.js",
+    "watch_dev_server": "NODE_ENV=dev nodemon --exec 'node server-tsc/server.js'",
+    "test_server": "yarn compile_server && yarn start_test_server & NODE_ENV=test yarn compile_tests && NODE_ENV=test mocha --opts mocha.opts",
```

Comment:
I still think we can find a better way to do this. We can only run this command once before it fails otherwise (beacuse it starts the server in a background thread).